server/models/profile.ts

```diff
@@ -0,0 +1,64 @@
+import * as Sequelize from 'sequelize';
+import { Model, DataTypes } from 'sequelize';
+
+import { UserAttributes, UserInstance } from './user';
+import { ModelStatic } from './types';
+import { AddressInstance } from './address';
+
+export interface ProfileAttributes {
+  id?: number;
+  user_id: number;
+  created_at?: Date;
+  updated_at?: Date;
+  profile_name?: string;
+  email?: string;
```

Comment:
Why do we need another email?

Reply:
@jalfredr

Reply:
The email in the User table is the "true" email, the single one associated with the user account for login, password retrieval, receipt of notifications, etc. The social media (including email) on the Profiles table is public-facing, permitting a user to put something (or nothing) visible to other users. I can definitely see a world where we split that off into another table downstream of Profile, but either way it should be distinct from the User version.

Reply:
okidoki. that makes sense to me, allowing people to expose a public email associated to their profile of multiple addresses